Source/itemdat.h

```diff
@@ -507,14 +507,14 @@ enum item_effect_type : int8_t {
 	IPL_INVCURS,
 	IPL_THORNS,
 	IPL_NOMANA,
-	IPL_NOHEALPLR, // unused
-	IPL_0x30,      // Unknown
-	IPL_0x31,      // Unknown
-	IPL_FIREBALL,  /* only used in hellfire */
-	IPL_0x33,      // Unknown
+	IPL_0x2F,     // unused
+	IPL_0x30,     // Unknown
+	IPL_0x31,     // Unknown
+	IPL_FIREBALL, /* only used in hellfire */
+	IPL_0x33,     // Unknown
 	IPL_ABSHALFTRAP,

```

Comment:
Alternatively we could give a fixed number to the next item in the list and then remove the unused ones.